tests/utils.go

```diff
@@ -204,25 +204,23 @@ func StopVirtualMachineWithTimeout(vm *v1.VirtualMachine, timeout time.Duration)
 	By("Stopping the VirtualMachineInstance")
 	virtClient := kubevirt.Client()
 
-	p, err := patch.New(
-		patch.WithAdd("/spec/running", nil),
-		patch.WithAdd("/spec/runStrategy", v1.RunStrategyHalted),
-	).GeneratePayload()
-	Expect(err).NotTo(HaveOccurred())
-	updatedVM, err := virtClient.VirtualMachine(vm.Namespace).Patch(context.Background(), vm.Name, types.JSONPatchType, p, metav1.PatchOptions{})
-	Expect(err).ToNot(HaveOccurred())
+	Expect(virtClient.VirtualMachine(vm.Namespace).Stop(context.Background(), vm.Name, &v1.StopOptions{})).To(Succeed())
```

Comment:
If the patch is replaced by the verb, can we drop the previous commit?

Reply:
Sure.